dotnet/src/SemanticKernel.Abstractions/Contents/StreamingChatMessageContent.cs

```diff
@@ -17,9 +19,53 @@ namespace Microsoft.SemanticKernel;
 public class StreamingChatMessageContent : StreamingKernelContent
 {
     /// <summary>
-    /// Text associated to the message payload
+    /// A convenience property to get or set the text of the first item in the <see cref="Items" /> collection of <see cref="StreamingTextContent"/> type.
     /// </summary>
-    public string? Content { get; set; }
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public string? Content
+    {
+        get
+        {
+            var textContent = this.Items.OfType<StreamingTextContent>().FirstOrDefault();
+            return textContent?.Text;
+        }
+        set
+        {
+            if (value is null)
+            {
+                return;
+            }
+
+            var textContent = this.Items.OfType<StreamingTextContent>().FirstOrDefault();
+            if (textContent is not null)
+            {
+                textContent.Text = value;
+                textContent.Encoding = this.Encoding;
+            }
+            else
+            {
+                this.Items.Add(new StreamingTextContent(
+                    text: value,
+                    choiceIndex: this.ChoiceIndex,
+                    modelId: this.ModelId,
+                    innerContent: this.InnerContent,
+                    encoding: this.Encoding,
+                    metadata: this.Metadata
+                ));
+            }
+        }
+    }
+
+    /// <summary>
+    /// Chat message content items.
+    /// </summary>
+    [JsonIgnore]
+    [Experimental("SKEXP0001")]
+    public StreamingKernelContentItemCollection Items
```

Comment:
Can we add an example how this property should be used?

Reply:
Yes, I have added them. Please check OpenAI_ChatCompletionStreaming.StreamTextContentAsync and OpenAI_ChatCompletionStreaming.StreamFunctionCallContentAsync.